properties.py

```diff
@@ -0,0 +1,79 @@
+import json
+import jsonschema
+import gi
+gi.require_version('Gst', '1.0')
+from gi.repository import GObject, Gst, GLib
+
+
+def load_from_json(pipeline, json_file='props.json', schema_file='props_schema.json'):
+    with open(json_file, 'r') as f:
+        properties = json.load(f)
+    with open(schema_file, 'r') as f:
+        schema = json.load(f)
+
+    try:
+        jsonschema.validate(properties, schema)
+    except jsonschema.exceptions.ValidationError as e:
+        print('not matching to schema', e)
+        return
+
+    for element in pipeline.children:
+        for prop in element.props:
+            try:
+                json_config = properties[element.name][prop.name]
+            except KeyError:
+                continue
+            setattr(element.props, prop.name, json_config)
+
+def dump_to_json(pipeline, json_file='props.json'):
+    elements = {}
+    for element in reversed(pipeline.children):
+        current_element = {}
+        for prop in element.props:
+            if 'Gst' in prop.value_type.name:
+                continue
+            current_element[prop.name] =  [prop.flags, 
+                                    prop.value_type.name, getattr(element.props, prop.name)]
+        elements[element.name] = current_element
+    print(elements)
+    with open(json_file, 'w') as f:
+        json.dump(elements, f, indent=4)
+
+def gprop_to_json(gprop, default_value):
```

Comment:
I think we can make this function more generic (LOL) by maybe keeping a dictionary that maps the string name like `'gboolean'` to a list of relevant `basic_prop`s that needs to be added and how to get it (if it is not too complicated). Maybe even create a generic class for like `JsonSchemeGstType` and we will invoke a class function like `to_json` of the relevant class according to the relevant `gprop_type`